### PR TITLE
Doc strings within 80 char limit for hmouse-tag.el

### PR DESCRIPTION
**Pull request by matsl**
## What

Doc strings within 80 char limit for hmouse-tag.el.

## Why

Warnings, warnings, warnings...

### CODE REVIEW

File: hmouse-tag.el [hunk 1]
```diff
@@ -573,12 +574,14 @@ in the current directory or any of its ancestor directories."
 
 ;;;###autoload
 (defconst smart-lisp-identifier-first-char-regexp "[-<>*a-zA-Z]"
-  "Regexp matching the first character of a Lisp identifier, including the square brackets.")
+  "Regexp matching the first character of a Lisp identifier.
```
Comment (rswgnu) [inline hmouse-tag.el:577]:
```suggestion
  "Character set regexp matching the first character of a Lisp identifier.")
```

File: hmouse-tag.el [hunk 2]
```diff
@@ -573,12 +574,14 @@ in the current directory or any of its ancestor directories."
...
+The square brackets are included.")
```
Comment (rswgnu) [inline hmouse-tag.el:578]:
```suggestion
```

File: hmouse-tag.el [hunk 3]
```diff
@@ -641,7 +644,9 @@ buffer."
     (smart-lisp-find-tag nil show-doc)))
 
 (defun smart-lisp-find-tag (&optional tag show-doc)
-  "Find the definition of optional Lisp TAG (or identifier at point when TAG is nil) or show its documentation with optional prefix arg SHOW-DOC non-nil.
+  "Find the definition of optional Lisp TAG or show its documentation.
+Use identifier at point when TAG is nil.  With optional prefix
+arg SHOW-DOC non-nil show its documentation.
```
Comment (rswgnu) [inline hmouse-tag.el:649]:
```suggestion
arg SHOW-DOC non-nil, show its documentation.
```

File: hmouse-tag.el [hunk 4]
```diff
@@ -688,7 +693,8 @@ Use `hpath:display-buffer' to show definition or documentation."
 		      (xref-etags-mode 0))))))))
 
 (defun smart-lisp-at-definition-p ()
-    "Return t when point is in a non-help buffer on the first line of a non-alias Lisp definition, else nil."
+  "Non-nil if point is on the first line of a non-alias Lisp definition.
```
Comment (rswgnu) [inline hmouse-tag.el:696]:
```suggestion
"Return t if in a non-help buf and on line 1 of a non-alias Lisp def, else nil."
```

File: hmouse-tag.el [hunk 5]
```diff
@@ -688,7 +693,8 @@ Use `hpath:display-buffer' to show definition or documentation."
...
+Returns nil if in a help buffer."
```
Comment (rswgnu) [inline hmouse-tag.el:697]:
```suggestion
```

File: hmouse-tag.el [hunk 6]
```diff
@@ -700,22 +706,24 @@ Use `hpath:display-buffer' to show definition or documentation."
 	     (not (looking-at "\\(;*[ \t]*\\)?(default"))))))
 
 (defun smart-lisp-at-load-expression-p ()
-    "Return t when point is on the first line of a Lisp library load expression, else nil."
+    "Non-nil if point is on the first line of a Lisp library load expression."
```
Comment (rswgnu) [inline hmouse-tag.el:709]:
```suggestion
"Return t if on the first line of a Lisp library load expression, else nil."
```

File: hmouse-tag.el [hunk 7]
```diff
@@ -700,22 +706,24 @@ Use `hpath:display-buffer' to show definition or documentation."
...
     (save-excursion
       (beginning-of-line)
       (looking-at "\\(;*[ \t]*\\)?(\\(autoload\\|load\\|require\\)\\s-")))
 
 (defun smart-lisp-at-change-log-tag-p ()
-  "When in a change-log mode, match to bound Elisp identifiers only and those with a '-' somewhere in the middle.
-These tight tests help eliminate undesired matches.
-Returns matching ELisp tag name that point is within, else nil."
+  "When in change-log mode match to bound Elisp identifiers only.
```
Comment (rswgnu) [inline hmouse-tag.el:715]:
```suggestion
  "When in change-log mode, match to bound Elisp identifiers only.
```

File: hmouse-tag.el [hunk 8]
```diff
@@ -700,22 +706,24 @@ Use `hpath:display-buffer' to show definition or documentation."
...
+Match to bound Elisp identifiers only and those with a '-'
```
Comment (rswgnu) [inline hmouse-tag.el:716]:
```suggestion
Also match to identifiers with a '-' in the middle.
```

File: hmouse-tag.el [hunk 9]
```diff
@@ -700,22 +706,24 @@ Use `hpath:display-buffer' to show definition or documentation."
...
+somewhere in the middle.  These tight tests help eliminate
```
Comment (rswgnu) [inline hmouse-tag.el:717]:
```suggestion
These tight tests help eliminate undesired matches.
```

File: hmouse-tag.el [hunk 10]
```diff
@@ -700,22 +706,24 @@ Use `hpath:display-buffer' to show definition or documentation."
...
+undesired matches.  Returns matching ELisp tag name that point is
```
Comment (rswgnu) [inline hmouse-tag.el:718]:
```suggestion
Return matching Elisp tag name that point is within, else nil."
```

File: hmouse-tag.el [hunk 11]
```diff
@@ -700,22 +706,24 @@ Use `hpath:display-buffer' to show definition or documentation."
...
+within, else nil."
```
Comment (rswgnu) [inline hmouse-tag.el:719]:
```suggestion
```

File: hmouse-tag.el [hunk 12]
```diff
@@ -875,12 +883,13 @@ If key is pressed:
...
   "Function that read a source FILE for jedi navigation.
-It takes these arguments: (file-to-read other-window-flag line_number column_number)."
+It takes these arguments: (file-to-read other-window-flag
+line_number column_number)."
   (hpath:display-buffer (find-file file) other-window)
   (jedi:goto--line-column line column))
 
 (defun smart-python-jedi-to-definition-p ()
-  "If the Jedi Python identifier server is running, test and use it to jump to the definition.
+  "If Jedi identifier server is running, test and use it to jump to the definition.
```
Comment (rswgnu) [inline hmouse-tag.el:892]:
```suggestion
  "If Jedi Python identifier server is running, use it to jump to the definition.
```

File: hmouse-tag.el [hunk 13]
```diff
@@ -1162,7 +1175,7 @@ TAG-SYM may be a function, variable or face."
       (error nil))))
 
 (defun smart-java-cross-reference ()
-  "If within a Java @see comment, displays the associated definition for editing and return non-nil, else nil.
+  "If within a Java @see comment display the associated definition for editing.
```
Comment (rswgnu) [inline hmouse-tag.el:1178]:
```suggestion
  "If within a Java @see comment, edit the def and return non-nil, else nil.
```

**Review by rswgnu — CHANGES_REQUESTED** [review]

**Comment by matsl** [comment]
@rswgnu Had to mix committing changes in the GUI with rebase and resolve some more complicated changes manually. 

When there are multi line changes I think it will be easier for me if all lines in the docstrings are changed within one suggested commit. I think that is possible by selecting multiple lines when creating the suggestion. Mark the fist line and use the mouse to drag to the last line you want to change.

File: hmouse-tag.el [hunk 14]
```diff
@@ -688,7 +692,7 @@ Use `hpath:display-buffer' to show definition or documentation."
 		      (xref-etags-mode 0))))))))
 
 (defun smart-lisp-at-definition-p ()
-    "Return t when point is in a non-help buffer on the first line of a non-alias Lisp definition, else nil."
+  "Non-nil if point is on the first line of a non-alias Lisp definition."
```
Comment (rswgnu) [inline hmouse-tag.el:695]:
```suggestion
  "Return non-nil if point is on the first line of a non-alias Lisp definition.
  Apply only to non-help buffers and return nil in others."
```

File: hmouse-tag.el [hunk 15]
```diff
@@ -1103,8 +1110,9 @@ Look for include file in `smart-c-cpp-include-path' and in directory list
       nil)))
 
 (defun smart-flash-tag (tag start end)
-  "Try to flash TAG at START to END in buffer, to indicate that it is serving as a hyperlink button.
-Returns TAG."
+  "Try to flash TAG at START to END in buffer.
+To indicate that it is serving as a hyperlink button.
+Return TAG."
```
Comment (rswgnu) [inline hmouse-tag.el:1115]:
```suggestion
  "Try to flash TAG at START to END in current buffer and return TAG.
This indicates that TAG is serving as a hyperlink button."
```

**Review by rswgnu — CHANGES_REQUESTED** [review]

File: hmouse-tag.el [hunk 16]
```diff
@@ -1114,7 +1122,9 @@ Returns TAG."
   tag)
 
 (defun smart-lisp-at-known-identifier-p ()
-  "Return non-nil if point is within a Lisp identifier listed in a tags table or is a known Emacs Lisp identifier, else nil."
+  "Non-nil if point is within a Lisp identifier.
+The Lisp identifier is either listed in a tags table or is a
+known Emacs Lisp identifier."
```
Comment (rswgnu) [inline hmouse-tag.el:1127]:
```suggestion
  "Return non-nil if point is within a known Lisp identifier.
The Lisp identifier is either listed in a tags table or is a
known Emacs Lisp identifier."
```

**Review by rswgnu — CHANGES_REQUESTED** [review]

**Review by rswgnu — APPROVED** [review]

**Pull request merged**